<?xml version="1.0"?>

<launch>

    <arg name="des_x" default= "-4"/>
    <arg name="des_y" default= "-3"/>
	<arg name="stato" default= "4"/>
    <param name="des_pos_x" value="$(arg des_x)" />
    <param name="des_pos_y" value="$(arg des_y)" />
	<param name="state" value="$(arg stato)" />
    <node pkg="final_assignment" type="wall_follow_service_m.py" name="wall_follower_switch" />
	<node pkg="final_assignment" type="randsix_pos_server.py" name="randsix_pos_server" />
    <node pkg="final_assignment" type="user_selection.py" name="user_selection" output="screen" />
    <node pkg="final_assignment" type="main.py" name="main" output="screen"/>
</launch>
